betty/locale.py

```diff
@@ -275,11 +280,27 @@ def negotiate_localizeds(preferred_locale: str, localizeds: List[Localized]) ->
 
 
 def open_translations(locale: str, directory_path: str) -> Optional[gettext.GNUTranslations]:
```

Comment:
Break BC and make this private. Then in the future we can consider compiling all layered translations into one.

Reply:
Let's not break BC here. We can always do that later if/when we've actually figured out how to aggregate translations and compile them into a single `*.mo` file.